controllers/queue_controller.go

```diff
@@ -127,6 +135,11 @@ func (r *QueueReconciler) addFinalizerIfNeeded(ctx context.Context, q *topologyv
 func (r *QueueReconciler) deleteQueue(ctx context.Context, client *rabbithole.Client, q *topologyv1alpha1.Queue) error {
 	logger := ctrl.LoggerFrom(ctx)
 
+	if client == nil {
+		logger.Info("Not deleting queue as RabbitmqCluster is already deleted or down", "queue", q.Name)
```

Comment:
I thought we are not deleting queues when the RabbitmqCluster is deleted/gone (when there is no secret object to generate a rabbithole client). What do you mean by the word `down` in sentence `RabbitmqCluster is already deleted or down`?

Reply:
This was down to my uncertainty about what 
```go
import client "sigs.k8s.io/controller-runtime/pkg/client"

client.Client{}.Get(ctx, types.NamespacedName{Name: rmq.Name, Namespace: rmq.Namespace}, cluster)
```
returning an error would signify. The Get returns an error if the cluster has been deleted, but are there other occasions where it would fail to find the object in the cache? Perhaps during a cluster restart? In which case, the object would be down but not deleted. I don't know if they can be distinguished between (but happy to be corrected!)